.circleci/ci_coverage.sh

```diff
@@ -26,4 +26,7 @@ EOF
 ./lcov-init.sh
 ctest -j20
 ./lcov-capture.sh coverage.info
+#lcov -r coverage.info -o coverage.info "*util*" "*trie*"  
```

Comment:
Seems this is a leftover from debugging?

Reply:
This removes certain folder from coverage.

Reply:
@ashtul I do not think we should remove trie and util from coverage, lets fix it.